### ISSUE TITLE
NTR 'natural capital'

### ISSUE THREAD

**Issue by mark-jensen** [SDG-InterfaceOntology/sdgio#94]
@pdez90 Jacquie requested this via email. Can you provide us some context for usage and definition? Or confirm the source below is acceptable.

I found this [here](http://www.naturalcapitaldeclaration.org/the-declaration/) 
> Natural capital comprises Earth’s natural assets (soil, air, water, flora and fauna), and the ecosystem services resulting from them, which make human life possible. Ecosystem goods and services from natural capital underpin productivity and the global economy. 

And this from the declaration linked on same page:
> The NCD defines natural capital as the stock of ecosystems that yields a renewable flow of goods and services that underpin the economy and provide inputs and direct and indirect benefits to businesses and society. Natural capital is a subset of environmental, social and governance (ESG) factors that can be material to financial institutions, mainly through their allocations of capital to companies through
loans and investments or premiums as part of insurance contracts.

**Comment by pbuttigieg** [comment]
Linked to discussion on #82

**Comment by pbuttigieg** [comment]
I believe this would be some sort of information artefact (perhaps a value specification?) and would have a definition that looks like:
`natural capital` =def. The total value of Earth's natural assets and the ecosystem services which depend on them.

I changed "provide" in the phrasing above (https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issue-134623639) as the asset need not be an ecosystem and thus cannot provide and ecosystem service, but can be part of an ecosystem and a necessary participant in an ecosystem service.

`natural asset` needs definition, see https://github.com/SDG-InterfaceOntology/sdgio/issues/82#issuecomment-182815616. @pdez90 could you provide one?
I would propose an initial definition: 
`natural asset` =def. An asset which is the result of or itself an environmental process that has not been substantially altered by human activity.

This definition suggests that an ecosystem service is, itself, a natural asset, which makes sense.

There are thresholding questions to address later: what's the boundary between natural and artificial or 'unnatural'? This is where a policy or declaration comes in.

We will eventually need `valuation process` to approach these classes more eloquently.

**Comment by pbuttigieg** [comment]
The Natural Capital Declaration definition above https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issue-134623639 seems to refer to the actual entities themselves, rather than some measure of value. 

They are using "capital" like:
>Capital is a type of good that can be consumed now, but if consumption is deferred, an increased supply of consumable goods is likely to be available later. (Wikipedia:[Capital_(economics)](https://en.wikipedia.org/wiki/Capital_%28economics%29))

rather than "Financial capital" as we saw with the discussions in #82 
>any form of wealth capable of being employed in the production of more wealth

Wikipedia's definition is helpful:
>Natural capital is the world's stock of natural resources, which includes geology, soils, air, water and all living organisms. Natural capital assets provide people with a wide range of free goods and services, often called ecosystem services, which underpin our economy and society and some of which even make human life possible.

Sourced from:
"What is natural capital?". naturalcapitalforum.com. World Forum on Natural Capital. Retrieved 31 December 2015.
and
"What is Natural Capital". www.naturalcapitalcoalition.org. Natural Capital Coalition. Retrieved 31 December 2015.

@mark-jensen 
Perhaps creating a class like "capital" which is the sum of [goods](https://en.wikipedia.org/wiki/Good_%28economics%29) where good=def. "A material entity which can be bought and sold in a market" OR resources (i.e. material entities with resource roles) would be useful. We can then have natural capital as "Capital composed of natural assets and the ecosystem services which depend on them."

**Comment by pbuttigieg** [comment]
@pdez90 we don't have any UNEP preferred definition at this stage. Please provide one.

**Comment by mark-jensen** [comment]
@pbuttigieg 
> The Natural Capital Declaration definition above #94 (comment) seems to refer to the actual entities themselves, rather than some measure of value.

Agree with this.

>Perhaps creating a class like "capital" which is the sum of goods where good=def. "A material entity which can be bought and sold in a market" OR resources (i.e. material entities with resource roles) would be useful. 

`capital` is odd, certainly role-like, closely tied to `resource` and `asset`. We will need to spend more time working the exact semantics out, fitting it all together. It is not immediately clear to me how to separate `asset` from `capital`. But for now:

`natural capital` = A material entity which is composed of the total of Earth's natural assets and the ecosystem services which depend on them

**Comment by pbuttigieg** [comment]
>capital is odd, certainly role-like, closely tied to resource and asset. We will need to spend more time working the exact semantics out, fitting it all together. It is not immediately clear to me how to separate asset from capital. 

I think this is a compositional class, a union operation over all material entities with asset roles. In this rendition, "capital" itself is a material entity.
@cmungall may know of a pattern that would work.

>natural capital = A material entity which is composed of the total of Earth's natural assets and the ecosystem services which depend on them

This may be better as a collection: 

`natural capital` = A collection of material entities which is composed of natural assets and the ecosystem services which depend on them.

I removed "Earth" as that would prevent subclassing. We can create a subclass like `global natural capital` to handle the Earth's total natural capital.

**Comment by pdez90** [comment]
Hi

I do not know what  discussions Jacquie had today at the meeting. I will
find out tomorrow and reply to all of the above

Thanks!!

On Thu, Feb 18, 2016 at 10:04 PM, Pier Luigi Buttigieg <
notifications@github.com> wrote:

> capital is odd, certainly role-like, closely tied to resource and asset.
> We will need to spend more time working the exact semantics out, fitting it
> all together. It is not immediately clear to me how to separate asset from
> capital.
>
> I think this is a compositional class, a union operation over all material
> entities with asset roles. In this rendition, "capital" itself is a
> material entity.
> @cmungall <https://github.com/cmungall> may know of a pattern that would
> work.
>
> natural capital = A material entity which is composed of the total of
> Earth's natural assets and the ecosystem services which depend on them
>
> This may be better as a collection:
>
> natural capital = A collection of material entities which is composed of
> natural assets and the ecosystem services which depend on them.
>
> I removed "Earth" as that would prevent subclassing. We can create a
> subclass like global natural capital to handle the Earth's total natural
> capital.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issuecomment-185863510>
> .
>



-- 
Yours Sincerely,
Priyanka deSouza

**Comment by mark-jensen** [comment]
@pbuttigieg Good, thanks!

**Comment by pbuttigieg** [comment]
I suggest that we are explicit and have 'natural capital' (strictly material goods / assets) and `natural capital and ecosystem services` (a union class with ecosystem services).

**Comment by mark-jensen** [comment]
`ecosystem services` belong in ENVO?

**Comment by pbuttigieg** [comment]
>ecosystem services belong in ENVO?
I'd say this will be in SDGIO, as the definition will rely on some regulatory specification (probably). I see quite a lot of webinars and papers on 'ecosystem services' as a contentious term. ENVO will host each `environmental process` that will be made an `ecosystem service` through a role or similar in SDGIO.

**Comment by pbuttigieg** [comment]
@mark-jensen 
My def needs some revision:

`natural capital` = A collection of entities which is composed of material natural assets and the ecosystem services which depend on them.

As we have processes in there, the collection contains more than just material entities.

**Comment by pdez90** [comment]
Hi, This term seems to be politically charged and countries havent yet agreed on the definition after much deliberation at the meeting at UNEP last week. 

I have attached a document circulated amongst countries by Jacquie. I will keep updating this thread with more information after speaking to her.

[International Definitions of Natural Capital and Natural Resources22.docx](https://github.com/SDG-InterfaceOntology/sdgio/files/140190/International.Definitions.of.Natural.Capital.and.Natural.Resources22.docx)

**Comment by pbuttigieg** [comment]
@pdez90 
Thanks for the update. We can represent alternative semantics with different labels and create a general class based on these (as long as they are not completely different entities). Please let us know when a more stable consensus is reached.

From the document you posted in https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issuecomment-187034714
>Natural capital are natural assets in their role of providing natural resource inputs and environmental services for economic production.

We can adapt the current definition of natural capital using the definition in the UNEP communique:
`natural capital` = A collection of material natural assets and the environmental services which depend on them.

The "economic production" aspect would be covered by the definition of "asset" and "natural asset" (see below). These make it clear that a material natural asset can serve as a natural resource in its own right (indeed, an asset is a subclass of resource). We could extend the definition (or add another, "official" definition annotation property) if desired, but this will be quite long winded.

This won't have the exact wording as the treatment of roles in ontology is very specific. @mark-jensen: We can include the 'official' wording in another annotation property called "official definition" or similar.

----

However:

We need a stable definition of natural asset. Right now we're considering:
>natural asset =def. An asset which is the result of or itself an environmental process that has not been substantially altered by human activity. https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issuecomment-185815834

The definition of `asset` was discussed in https://github.com/SDG-InterfaceOntology/sdgio/issues/82#issuecomment-182815616 and https://github.com/SDG-InterfaceOntology/sdgio/issues/82#issuecomment-184310872. But we still need some input to finalise it. It would be something along the lines of: 

`asset`= A resource which is determined to have positive economic value, is owned or otherwise controlled by a person or organisation, has the disposition to increase the wealth of its owner, and which can be exchanged in an economic system for other assets.
* A country can be an organisation
* "economic value" would be the information output of a `economic valuation process`
* "other assets' includes cash
* land, ecosystems, and natural resource stocks can all have this role, so it will be semantically consistent with the context of the UNEP communique. 
* @mark-jensen logically, this would be any entity that has an `asset role`, where the `asset role` definition would be derived from that of asset, above.
* the types of assets would be closely tied to the types of capital (either the collection of assets or the sum of the [monetary] value of the assets)
* tangible asset and intangible asset would be subclasses that may be filled by inference. 

This draws from the [IFRS definition](http://www.iasb.org/NR/rdonlyres/1C268B9D-C477-4C0F-BA86-7C540ACA5E56/0/Dec050512ob02a.pdf) quoted in [Wikipedia](https://en.wikipedia.org/wiki/Asset):
>One of the most widely accepted accounting definitions of asset is the one used by the International Accounting Standards Board.[6] The following is a quotation from the IFRS Framework: "An asset is a resource controlled by the enterprise as a result of past events and from which future economic benefits are expected to flow to the enterprise."
This means that:
>
* The probable present benefit involve a capacity, singly or in combination with other assets, in the case of profit oriented enterprises, to contribute directly or indirectly to future net cash flows, and, in the case of nonprofit organizations, to provide services;
* The entity can control access to the benefit;
* The transaction or event giving rise to the entity's right to, or control of, the benefit has already occurred.

**Comment by pbuttigieg** [comment]
On natural resources:
> Definition:
Natural resources are natural assets (raw materials) occurring in nature that can be used for economic production or consumption.
>
Context:
The naturally occurring assets that provide use benefits through the provision of raw materials and energy used in economic activity (or that may provide such benefits one day) and that are subject primarily to quantitative depletion through human use. They are subdivided into four categories: mineral and energy resources, soil resources, water resources and biological resources.
> United Nations, European Commission, International Monetary Fund, Organisation for Economic Co-operation and Development, World Bank, 2005, Handbook of National Accounting: Integrated Environmental and Economic Accounting 2003, Studies in Methods, Series F, No.61, Rev.1, Glossary, United Nations, New York, para. 7.42, EA.1


This would invert the semantics we are currently discussing: a given asset as a type of resource (see https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issuecomment-187746261). This makes more sense to me, as all resources are not necessarily economic assets, but all economic assets are clearly resources. See #91, which argues that "raw materials" are simply a subset of "natural assets" (they have to be material entities). We can further specify that these must belong to the four categories quoted above.

If the deliberating parties insist on this definition, we'll have to discuss this more to 'decode' the various perspectives around these terms and find out what the underlying semantics are.

**Comment by mark-jensen** [comment]
>We can include the 'official' wording in another annotation property called "official definition" or similar.

For now I will leave it as an editor note. Something like:
`natural capital` =offontodef "A collection composed of material entities bearing natural asset roles and the environmental services which they are inputs for"

@pbuttigieg  Do I have that right?

**Comment by pbuttigieg** [comment]
No, we wouldn't be generating the official defs. They would be verbatim from sanctioned docs. 

Also, is there a distinction intended between environmental service and ecosystem service? While dealing with those parts of the planet that humans inhabit, these are interchangeable terms (I.e. both class definitions hold)

**Comment by pdez90** [comment]
Do you need me to ask about the difference between environment and
ecosystem services?

On Thu, Feb 25, 2016 at 9:49 AM, Pier Luigi Buttigieg <
notifications@github.com> wrote:

> No, we wouldn't be generating the official defs. They would be verbatim
> from sanctioned docs.
>
> Also, is there a distinction intended between environmental service and
> ecosystem service? While dealing with those parts of the planet that humans
> inhabit, these are interchangeable terms (I.e. both class definitions hold)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issuecomment-188639629>
> .
>



-- 
Yours Sincerely,
Priyanka deSouza

**Comment by pbuttigieg** [comment]
@pdez90 
>Do you need me to ask about the difference between environment and ecosystem services?

Yes, this would be helpful. I see no real difference right now, but there may be some definitional subtlety I'm not familiar with

@mark-jensen 
To clarify my response above (from a smartphone) https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issuecomment-188639629:
This def:
natural capital ="A collection composed of material entities bearing natural asset roles and the environmental services for which they are inputs"

Would be the 'normal' ontology definition (except we'd have a defined class for `natural asset` to avoid having roles in the text def). There would then be an "official definition" or similar (we wouldn't use abbreviations here) like the ones you found initially https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issue-134623639, assuming they are adopted:

>>Natural capital comprises Earth’s natural assets (soil, air, water, flora and fauna), and the ecosystem services resulting from them, which make human life possible. 

...
>>the stock of ecosystems that yields a renewable flow of goods and services that underpin the economy and provide inputs and direct and indirect benefits to businesses and society.

**Comment by pbuttigieg** [comment]
Perhaps of interest for [measuring natural capital](http://climatenewsnetwork.net/scientists-calculate-our-debt-to-the-earth/)

**Comment by mark-jensen** [comment]
> There would then be an "official definition" or similar (we wouldn't use abbreviations here) like the ones you found initially #94 (comment), assuming they are adopted:

'official' in the sense of "official UN". Got it. Thus we will have 'UNEP preferred label' and 'official definition' APs. We will need to confirm this works for UNEP and that they understand our reasoning, and then of course decide how this will be implemented in the release version, visualized on the portal.

**Comment by mark-jensen** [comment]
@pbuttigieg 

`natural capital` = A collection composed of natural assets and the environmental services for which they are inputs.

How's that? Using 'input' vs 'depend', but that may change intended meaning.

**Comment by pbuttigieg** [comment]
>How's that? Using 'input' vs 'depend', but that may change intended meaning. 

Yes, I wouldn't do this: the material entity could be any form of participant rather than an input. Removing it could still kill the ecosystem/evolutionary service.
